### PR TITLE
fix: Issue 181

### PR DESCRIPTION
**Pull request by balloonpopper**
Updates some documentation along with room changes to demonstrate usage of commands without examples
Fixes https://github.com/godot-escoria/escoria-issues/issues/181

### CODE REVIEW

File: addons/escoria-core/game/core-scripts/esc/commands/set_angle.gd [hunk 1]
```diff
@@ -1,17 +1,35 @@
...
+# degrees, a 4 direction character only 2. The wait time will determine how
+# long the idle animation for each direction is played before using the next
+# direction's animation. As such, if wait was set to 1 second, a 16 direction
+# character would take 8 seconds to turn 180 degrees, a 4 direction character
+# would take 2 seconds.
 #
 # **Parameters**
 #
 # - *object*: Global ID of the object to turn
 # - *degrees*: Number of degrees by which `object` is to be turned
```
Comment (BHSDuncan) [inline addons/escoria-core/game/core-scripts/esc/commands/set_angle.gd:29]:
```suggestion
# - *target_degrees*: Number of degrees by which `object` is to be turned
```

File: addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd [hunk 2]
```diff
@@ -1,12 +1,19 @@
-# `stop_snd [player]`
+# `stop_snd [audio bus]`
 #
-# Stops the given sound player's stream.
+# Stops the given audio buses stream.
```
Comment (BHSDuncan) [inline addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd:3]:
```suggestion
# Stops the given audio bus's stream.
```

File: addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
 #
+# By default there are 3 audio busses set up by Escoria : `_sound`, which is
```
Comment (BHSDuncan) [inline addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd:5]:
```suggestion
# By default there are 3 audio buses set up by Escoria : `_sound`, which is
```

File: addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+# Each simultaneous sound (e.g. multiple game sound effects) will require its 
+# own bus. To create additional buses, see the Godot sound documentation :
+# [Audio buses](https://docs.godotengine.org/en/stable/tutorials/audio/audio_buses.html#doc-audio-buses)
+# 
 # **Parameters**
 #
-# - *player*: Sound player to use. Either `_sound`, which is used to play non-
-#   looping sound effects; `_music`, which plays looping music; or `_speech`, which
-#   plays non-looping voice files (default: `_music`)
+# - *audio bus*: Bus to stop ("_sound", "_music", "_speech", or a custom
```
Comment (BHSDuncan) [inline addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd:15]:
```suggestion
# - *audio_bus*: Bus to stop ("_sound", "_music", "_speech", or a custom
```

**Review by BHSDuncan — CHANGES_REQUESTED** [review]
Looks good! Just a few small-ish fixes.

Also, I see that there are two shader materials that are a part of this PR. Why is that?

**Review by BHSDuncan — APPROVED** [review]

**Review by StraToN — APPROVED** [review]

**Pull request merged**